website/project/model.py

```diff
@@ -2977,6 +3009,9 @@ def add_contributor(self, contributor, permissions=None, visible=True,
                 while len(user.recently_added) > MAX_RECENT_LENGTH:
                     user.recently_added.pop()
 
+            if contrib_to_add.is_registered and settings.ENABLE_NOTIFICATION_SUBSCRIPTION_CREATION:
```

Comment:
From @sloria on the previous PR:

>The `is_registered` check should probably be moved to the `subscribe_user_to_notifications` method so that the caller doesn't have to do this check every time.